examples/image_classification/image_classification_timm_peft_lora.ipynb

```diff
@@ -195,7 +195,7 @@
     }
    ],
    "source": [
-    "ds = load_dataset('beans')"
+    "ds = load_dataset(\"beans\")"
```

Comment:
Could you please undo the changes in this and the other notebook? I think they are probably added accidentally and have nothing to do with this PR.

Reply:
The changes were made when `make style && make quality` was run. Should not cause a problem right?

Reply:
Yes, I guess it should be okay this time.

Ideally we want to avoid completely unrelated changes, as that makes the code harder to review and commits more difficult to revert. Here I guess what happened is that you have installed the jupyter extension for black, which resulted in these files being updated.